flatpak/org.DolphinEmu.dolphin-emu.metainfo.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2016 Jeremy Newton -->
```

Comment:
Jeremy Newton is the maintainer of the fedora package from where this file was originally copied back in 2017, It's very likely that it has been entirely rewritten since then.

Changing the copyright is unlikely to be an issue if need be.